docs/release.md

```diff
@@ -0,0 +1,68 @@
+# OSA Release process
+
+OSA Release process consist of multiple dimensions:
+* Plugin code release
+* Config release
+* VM image release
+
+Update might be one or any combination of all dimensions.
```

Comment:
I assume that this is not totally true, as in we *may* not support upgrading between openshift 3.x and 4.x (so how do we restrict this). we should probably assume that the vm images and config are at the same major version as the plugin code - right/no?
also are we going to use the same versioning scheme between the 3 entities or are the versions unrelated? (just really asking you to be specifc) - is the version based off of the openshift version or independent? - given that the addons are probably bound to a specifc openshift version it would make sense these are the same and related to the openshift version. arguably we could place the stuff that runs within the cluster in a seperate repo (sync pod/controller/backup and addons)

Reply:
> we should probably assume that the vm images and config are at the same major version as the plugin code - right/no? 

Not sure about this. I think we can version plugin code separately, so if we release fast we will not end up with `3.11.something.354` 

> also are we going to use the same versioning scheme between the 3 entities or are the versions unrelated? (just really asking you to be specifc) - is the version based off of the openshift version or independent? - given that the addons are probably bound to a specifc openshift version it would make sense these are the same and related to the openshift version. arguably we could place the stuff that runs within the cluster in a seperate repo (sync pod/controller/backup and addons)

We should lock config version with VM version, but plugin code versioning might be different and independent. 

Do you have versioning standard in mind?